scripts/install

```diff
@@ -43,14 +43,24 @@ chmod 644 "/etc/cron.d/$app"
 #=================================================
 # ADD A CONFIGURATION
 #=================================================
-ynh_script_progression --message="Adding a configuration file..." --weight=5
+ynh_script_progression --message="Adding a configuration files..." --weight=5
 
 # Install configuration files
 ynh_add_config --template="config.local.php" --destination="$install_dir/config.local.php"
 
-chmod 650 "$install_dir/config.local.php"
+chmod 440 "$install_dir/config.local.php"
 chown $app:$app "$install_dir/config.local.php"
 
+ynh_script_progression --message="Adding a configuration directory..." --weight=5
+mkdir "$data_dir/data"
+chmod 770 "$data_dir/data"
+chown -R $app:www-data "$data_dir/data"
+
+ynh_script_progression --message="Adding a configuration file..." --weight=5
+ynh_add_config --template="config.local.user.php" --destination="$data_dir/data/config.local.user.php"
+chmod 440 "$data_dir/data/config.local.user.php"
```

Comment:
For this one, `640`, as the user and root have to be able to edit it.

Reply:
I keep as before the install process without trying add the file, it is source of mistakes as you see before. The file will be added after a first upgrade...

Reply:
Ok

Reply:
Is it OK for you like this ?